_posts/2020-12-02-spack-user-survey-2020.md

```diff
@@ -0,0 +1,918 @@
+the 7th most popular application area outside ECP, but the 4th most
+popular inside ECP. Only one user reported using Spack for web
+applications.
+
+## How did you find out about Spack?
+
+<figure>
+  <a href="/assets/images/spack-user-survey-2020/two_pies_how_find_out.svg">
+    <img src="/assets/images/spack-user-survey-2020/two_pies_how_find_out.svg">
+  </a>
+</figure>
+
+Both in and outside ECP, about half of Spack users hear about the tool
+via word of mouth. This result made us pretty happy -- we think it means
+that users are very willing to recommend Spack to friends. After word of
+mouth, 22% people heard of Spack because it was used at their site.
+Within ECP this was slightly more likely at 30%.
+
+24% of users heard about Spack from outreach activities: tutorials, BOF
+sessions, and presentations.
+
+
+## How long have you been using Spack?
+
+<figure>
+  <a href="/assets/images/spack-user-survey-2020/two_bars_how_long_using.svg">
+    <img src="/assets/images/spack-user-survey-2020/two_bars_how_long_using.svg">
+  </a>
+</figure>
+
+Spack usage has been growing over time, and the number of users who join
+the community each year is increasing. Most of the overall community has
+been using Spack for less than two years. We also see this effect in the
+number of contributors to the project on GitHub. In 2018, after the
+project had been publicly available for 4 years, there were around 300
+contributors. 2 years later, there are nearly 700.
+
+Within ECP, the community is older -- most people started using Spack in
+ECP 2-3 years ago. There has been less adoption since then, but we
+attribute this to Spack's rapid adoption in ECP. Spack caught on quickly
+there, people have continued to use it, and there is not a large influx
+of users *into* ECP. The population stays mostly the same over time.
+
+## Have you contributed to Spack?
+
+<figure>
+  <a href="/assets/images/spack-user-survey-2020/two_multi_bars_how_contributed.svg">
+    <img src="/assets/images/spack-user-survey-2020/two_multi_bars_how_contributed.svg">
+  </a>
+</figure>
+
+Around 75% of users who responded have contributed to Spack in some way
+or another, and most of those (60%) have contributed a package. Nearly
+40% of users are active on Slack, and nearly 40% have filed issues on
+GitHub.
+
+It's notable that Slack discussions are *far* more active than the Spack
+mailing list -- users seem to want to engage live rather than sending
+emails back and forth.
+
+Not many users (~10%) have contributed to documentation, but even this
+small amount helps the project -- this is around 16 people.
+
+There don't seem to be any special takeaways here for ECP, except that
+ECP users seem to be slightly more likely to contribute a package (nearly
+70% have done so).
+
+# Spack Usage
+
+Having characterized the user base, we moved on to finding out how they
+use Spack.
+
+## What version(s) of Spack do you use?
+
+<figure>
```

Comment:
Is there a meaningful difference between "not sure" and "do not know"? Should those responses be combined in the graph?

Reply:
good point -- fixed